apps/fishing-map/features/analysis/Analysis.tsx

```diff
@@ -226,16 +226,16 @@ function Analysis() {
             disabled={!bounds}
           />
         </div>
+        <div>
+          <p className={styles.placeholder}>
```

Comment:
this was hidden on purpose when the button is not visible as it says: `Click the button below if you need a more precise anlysis` and is confusing.
If we always want to show `The data shown above should be taken as an estimate.` then we should split the message

Reply:
ok, will do